feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -219,7 +218,7 @@ fun ForYouScreen(
                     )
 
                     item {
-                        Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
```

Comment:
We don't want to remove the insets support here, as otherwise cards could be obscured by the bottom insets when we're using the navigation rail instead of the navigation bar.

If we add another `Spacer` with the `8.dp` height that should fix the issue and avoid overlapping with insets!

Reply:
Done